config/software/bs4-python.rb

```diff
@@ -0,0 +1,11 @@
+name "beautifulsoup4"
+default_version "4.5.1"
+
+
+dependency "python"
+dependency "pip"
+
+build do
+  ship_license "MIT"
```

Comment:
should we include the license from the repo (with the copyright and the custom mentions) from [there](http://bazaar.launchpad.net/~leonardr/beautifulsoup/bs4/view/head:/LICENSE) instead of a generic MIT licence?